src/helpers/authHelper.js

```diff
@@ -0,0 +1,43 @@
+const { secret, tokens } = require('../../config/jwtConfig').jwtData;
+const tokenList = require('../models/TokenModel');
+const jwt = require('jsonwebtoken');
+import { v4 as uuidv4 } from 'uuid';
+
+const generateAccessToken = (userId) => {
```

Comment:
@lenon777, Bcrypt/JWT services would be the best place for these helper methods.